MAINTAINERS.yml

```diff
@@ -439,7 +461,13 @@ Documentation:
         - "area: Crypto / RNG"
 
 "Drivers: ESPI":
-    status: orphaned
+    status: maintained
+    maintainers:
+      - albertofloyd
+    collaborators:
+      - VenkatKotakonda
+      - scottwcpg
+      - albertofloyd
```

Comment:
This lists @albertofloyd as both maintainer and collaborator?

Reply:
no harm I guess, a maintainer is still a collab